packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -206,6 +206,7 @@ class SemanticsData with Diagnosticable {
     this.tags,
     this.transform,
     this.customSemanticsActionIds,
+    required this.editableTextId,
```

Comment:
Should this only be set if the flags indicate that it is a textfield? Maybe assert?

Reply:
Unfortunately, const asserts don't support the expression `flag & SemanticsFlag.isTextField.value == 0`. If you can think of a workaround, I'm all ears. FWIW, I'll have this assertion in `dart:ui`.